packages/paste-website/src/pages/components/blockquote/index.mdx

```diff
@@ -0,0 +1,192 @@
+import {Blockquote, BlockquoteContent, BlockquoteCitation} from '@twilio-paste/blockquote';
+import packageJson from '@twilio-paste/blockquote/package.json';
+import { ExampleText } from '@twilio-paste/example-text';
+import {CodeBlock} from '@twilio-paste/code-block'
+import {InlineCode} from '@twilio-paste/inline-code'
+import {Label} from '@twilio-paste/label'
+import {Input} from '@twilio-paste/input'
+import {HelpText} from '@twilio-paste/help-text'
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout'
+import {Box} from "@twilio-paste/box"
+import {Paragraph} from "@twilio-paste/paragraph"
+
+import {SidebarCategoryRoutes} from '../../../constants';
+import ComponentPageLayout from '../../../layouts/ComponentPageLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+
+export const meta = {
+  title: 'Blockquote - Components',
+  package: '@twilio-paste/blockquote',
+  description: packageJson.description,
+  slug: '/components/blockquote/',
+};
+
+export default ComponentPageLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Blockquote');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+      mdxHeadings,
+      pageHeaderData: {
+        categoryRoute: SidebarCategoryRoutes.COMPONENTS,
+        githubUrl: 'https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/blockquote',
+        storybookUrl: '/?path=/story/components-blockquote--default',
+      },
+    },
+  };
+};
+
+<LivePreview
+  scope={{Blockquote, BlockquoteContent, BlockquoteCitation}}
+  language="jsx"
+>
+  {`<Blockquote url="#">
+      <BlockquoteContent>
+        With AI-driven products, the design process is no longer just about aesthetics. It’s about designing for the
+        human experience as a whole.
+      </BlockquoteContent>
+      <BlockquoteCitation author="Google" source="People + AI Guidebook" />
+    </Blockquote>`}
```

Comment:
nit: the indentation here looks off in the live preview code block. First line fine but need to move rest of content closer to start